src/gtfs/read.rs

```diff
@@ -631,6 +631,26 @@ fn manage_comment_from_stop(
     comment_links
 }
 
+fn manage_comment_from_route(
+    comments: &mut CollectionWithId<objects::Comment>,
+    route: &Route,
+) -> CommentLinksT {
```

Comment:
I know we have a lot of examples of non-pure functions in the code but do you think it'd be possible to have a pure-function here that may only return a `Option<Comment>`? Something like this.

```rust
fn get_route_comment(route: &Route) -> Option<Comment> {
   // omitted
}
```

Reply:
Way nicer!
I’m adding a commit to handle the same way the stop comments (I just copied-paste…)